src/app/views/provisioning/confirm.html

```diff
@@ -22,7 +22,7 @@
           <td>{{ vm.subscription.product_pricing.name }}</td>
           <td>{{ vm.subscription.product_pricing.description }}</td>
           <td>
-            <span ng-hide="PRICING_TYPES['unpriced'].indexOf(pricingPlan.pricing_type) != -1" ng-repeat="price in vm.subscription.product_pricing.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
+            <span ng-if="vm.pricedPlan(vm.subscription.product_pricing)" ng-repeat="price in vm.subscription.product_pricing.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
```

Comment:
I'd use `ng-show` here instead as that is what's used throughout this file.

Reply:
Also I'd only use `ng-if` if for some reason we need the DOM element to be completely removed from the page. `ng-hide` & `ng-show` are usually going to be sufficient (they just affect styling).

Reply:
@adamaziz15 I am modelling after https://github.com/maestrano/mno-enterprise-angular/pull/433/files -- should I also change these to ng-hide ?

Reply:
@iseessel Yes please, in general I think `ng-hide`/`ng-show` should be used together. It's a bit safer as well as there could be implications to using `ng-if` - it not only removes elements from the DOM, it also creates a child scope.